repositories.py

```diff
@@ -119,4 +119,11 @@
         "<minitwit_url>",
         "<minitwit_api_url>",
     ],
+    [
+        "group s",
+        "Group S",
+        ["https://github.com/ingrid-mc/DevOps-group_s", ],
+        "https://github.com/ingrid-mc/DevOps-group_s/tree/main/itu-minitwit",
```

Comment:
Leave this line with `<minitwit_url>` for now and keep the github link on the line above. You will place a link to your live server here later.